lib/index.js

```diff
@@ -192,13 +200,11 @@ class Configurator {
 		// TODO: these modules need to be transpiled
 		// NOTE: these are npm modules, babel has a freak out if you just include all of /node_modules/
 		const es6Modules = [
-			'@financial-times/n-card',
-			'@financial-times/n-image',
-			'@financial-times/n-section',
-			'@financial-times/n-ui'
+			'@financial-times/n-'
```

Comment:
but only n- npm modules that are required by the client side code... which is never gonna be that big a list and I can't see that we'd want any such modules to be excluded from babel